apps/documentation/src/routes/documentation/tutorial/redirections.mdx

```diff
@@ -13,13 +13,11 @@ import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
 # Redirections
 
-## Handle redirections
+What if there is a Pokémon among all of them that should be considered the GOAT? What
+we are going to do right now is create a new route `/pokemons/GOAT` that points to the best
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Consider using either the past participle “created” or the present participle “creating” here. (BEEN_PART_AGREEMENT[9])
Suggestions: `created`, `creating`
URL: https://languagetool.org/insights/post/verb-tenses/ 
Rule: https://community.languagetool.org/rule/show/BEEN_PART_AGREEMENT?lang=en-US&subId=9
Category: GRAMMAR
<!-- __reviewdog__:ChBlYjMwYmM0YjJjMWQ4NzEzEgxMYW5ndWFnZVRvb2w= -->